src/sentry_plugins/asana/plugin.py

```diff
@@ -202,7 +203,14 @@ def get_config(self, *args, **kwargs):
             client = self.get_client(user)
         except PluginIdentityRequired as e:
             self.raise_error(e)
-        workspaces = client.get_workspaces()
+        try:
+            workspaces = client.get_workspaces()
+        except Exception as e:
```

Comment:
@ceorourke do we know the specific type of error we expect here? That would be preferable to catching all errors.